README.md

```diff
@@ -213,71 +284,24 @@ module "landing_zone" {
   }
 ```
 
-## Enable SNS topic subscription
+## SNS topic subscription
+
+| Topic Name  |  Variable | Content  |
+|---|---|---|
+| `aws-controltower-AggregateSecurityNotifications` | `sns_aws_config_subscription`  |  Aggregated AWS Config notifications |
```

Comment:
I feel like I've asked this before, but why is this `aws-controller` and below `LandingZone`?

Reply:
I dont understand the question :-D Can you eleborate?

Reply:
Ah you mean why aws-controltower and not LandingZone? Because this is a default topic created by aws-controltower and the other 2 are custom :-)